examples/synthetics_mesh_example_test.go

```diff
@@ -288,24 +288,29 @@ func toPercent(v float64) float64 {
 	return v * 100
 }
 
-func formatMetricData(md *syntheticspb.MetricData) string {
+func formatMetricData(md *syntheticspb.MetricData, isCurrentMeasurementValid bool) string {
 	if md == nil {
 		return "[X]\t"
 	}
 
 	return fmt.Sprintf(
 		"%v / %v / %v / %v\t",
-		formatMetricValue(md.GetCurrent()),
-		formatMetricValue(md.GetRollingAvg()),
-		formatMetricValue(md.GetRollingStddev()),
+		formatMetricValue(md.GetCurrent(), isCurrentMeasurementValid),
+		formatMetricValue(md.GetRollingAvg(), isCurrentMeasurementValid),
+		formatMetricValue(md.GetRollingStddev(), isCurrentMeasurementValid),
 		md.GetHealth(),
 	)
 }
 
 // formatMetricValue formats the value of metric given in nanoseconds to milliseconds.
-func formatMetricValue(metricValue uint32) string {
-	if metricValue == 0 {
+func formatMetricValue(metricValue uint32, isCurrentMeasurementValid bool) string {
+	if metricValue == 0 && !isCurrentMeasurementValid {
```

Comment:
Current measurement invalid alone does not suffice to mark the measurement as [X] ?

Reply:
I does suffice for latency and jitter current values. But not for latency and jitter rolling average / stddev. This case happens in Petah Tikva -> Zhangjiakou connection. Current packet loss is 100%, but averages and stddev are valid measurements.

Reply:
Full log with packet loss:

```
Picked network_mesh test named "network_mesh_6x6" with ID 6232
Number of test results: 56
Latest test results time: 2022-04-07 15:00:00 +0000 UTC
Table cells contain ping latency and its health in format: "current [ms] / rolling avg. [ms] / rolling stddev. [ms] / health"
.                             Cape Town, South Africa      Chicago, IL, United States    Osaka, Japan                 Zhangjiakou, China          Petah Tikva, Israel         Tallinn, Estonia           
Cape Town, South Africa       [X]                          250 / 244 / 3 / healthy       394 / 392 / 6 / healthy      304 / 291 / 4 / healthy     211 / 218 / 3 / healthy     179 / 179 / 2 / healthy    
Chicago, IL, United States    [X] / [X] / [X] / failing    [X]                           [X] / [X] / [X] / failing    222 / 226 / 12 / healthy    147 / 148 / 3 / healthy     123 / 124 / 0 / healthy    
Osaka, Japan                  394 / 394 / 0 / healthy      187 / 220 / 26 / healthy      [X]                          86 / 125 / 37 / healthy     275 / 325 / 19 / healthy    292 / 291 / 2 / healthy    
Zhangjiakou, China            [X]                          224 / 221 / 3 / healthy       [X] / [X] / [X] / failing    [X]                         283 / 300 / 12 / healthy    267 / 269 / 4 / healthy    
Petah Tikva, Israel           [X]                          147 / 148 / 3 / healthy       [X]                          [X] / 296 / 9 / critical    [X]                         93 / 91 / 3 / healthy      
Tallinn, Estonia              [X]                          123 / 124 / 0 / healthy       [X]                          267 / 261 / 10 / healthy    92 / 84 / 5 / healthy       [X]                        

Table cells contain ping jitter and its health in format: "current [ms] / rolling avg. [ms] / rolling stddev. [ms] / health"
.                             Cape Town, South Africa      Chicago, IL, United States    Osaka, Japan                 Zhangjiakou, China        Petah Tikva, Israel    Tallinn, Estonia       
Cape Town, South Africa       [X]                          0 / 0 / 0 / healthy           0 / 0 / 0 / healthy          8 / 1 / 2 / warning       1 / 0 / 0 / healthy    0 / 0 / 0 / healthy    
Chicago, IL, United States    [X] / [X] / [X] / failing    [X]                           [X] / [X] / [X] / failing    1 / 2 / 2 / healthy       1 / 0 / 0 / healthy    0 / 0 / 0 / healthy    
Osaka, Japan                  0 / 0 / 0 / healthy          0 / 0 / 0 / healthy           [X]                          0 / 1 / 3 / healthy       0 / 0 / 0 / healthy    0 / 0 / 0 / healthy    
Zhangjiakou, China            [X]                          2 / 2 / 2 / healthy           [X] / [X] / [X] / failing    [X]                       0 / 1 / 1 / healthy    0 / 0 / 0 / healthy    
Petah Tikva, Israel           [X]                          1 / 0 / 0 / healthy           [X]                          [X] / 1 / 1 / critical    [X]                    0 / 0 / 0 / healthy    
Tallinn, Estonia              [X]                          0 / 0 / 0 / healthy           [X]                          0 / 2 / 2 / healthy       0 / 0 / 0 / healthy    [X]                    

Table cells contain ping packet loss and its health in format: "current [%] / health"
.                             Cape Town, South Africa    Chicago, IL, United States    Osaka, Japan          Zhangjiakou, China     Petah Tikva, Israel    Tallinn, Estonia    
Cape Town, South Africa       [X]                        0.00 % / healthy              0.00 % / healthy      0.00 % / healthy       0.00 % / healthy       0.00 % / healthy    
Chicago, IL, United States    100.00 % / failing         [X]                           100.00 % / failing    40.00 % / healthy      0.00 % / healthy       0.00 % / healthy    
Osaka, Japan                  0.00 % / healthy           0.00 % / healthy              [X]                   0.00 % / healthy       0.00 % / healthy       0.00 % / healthy    
Zhangjiakou, China            [X]                        0.00 % / healthy              100.00 % / failing    [X]                    0.00 % / healthy       0.00 % / healthy    
Petah Tikva, Israel           [X]                        0.00 % / healthy              [X]                   100.00 % / critical    [X]                    0.00 % / healthy    
Tallinn, Estonia              [X]                        0.00 % / healthy              [X]                   20.00 % / healthy      0.00 % / healthy       [X]                 

--- PASS: TestDemonstrateSyntheticsNetworkMeshTestResults (5.15s)
PASS

Process finished with the exit code 0

```